controllers/user.js

```diff
@@ -43,6 +45,8 @@ export const update = async interaction => {
   const score = calculateScore(interaction);
   const userInfo = await getUserInfo(interaction.user);
 
+  // isCoreTeam(interaction.user);
```

Comment:
I think it would be nice to remove this comment

Reply:
Yes! Thanks for notify.